scylla/src/transport/load_balancing/default.rs

```diff
@@ -636,8 +636,7 @@ impl Default for DefaultPolicy {
 /// # }
 #[derive(Clone, Debug)]
 pub struct DefaultPolicyBuilder {
-    preferred_datacenter: Option<String>,
-    preferred_rack: Option<String>,
+    preferences: ReplicaLocationPreference,
```

Comment:
The message of the last commit contains:

`in <insert_commit_hash>`

Which commit hash? I don't know which commit this refers to.

Reply:
Good catch. This commit is a WIP and is intented for the future major release, because it contains API-breaking changes. I pushed it by accident.